erpnext/accounts/report/financial_statements.py

```diff
@@ -13,86 +13,160 @@
 from erpnext.accounts.report.utils import get_currency, convert_to_presentation_currency
 from erpnext.accounts.utils import get_fiscal_year
 from frappe import _
-from frappe.utils import (flt, getdate, get_first_day, add_months, add_days, formatdate, cstr)
+from frappe.utils import (flt, getdate, add_months, add_days, formatdate, get_last_day, cstr)
 
 from six import itervalues
 from erpnext.accounts.doctype.accounting_dimension.accounting_dimension import get_accounting_dimensions, get_dimension_with_children
 
-def get_period_list(from_fiscal_year, to_fiscal_year, periodicity, accumulated_values=False,
-	company=None, reset_period_on_fy_change=True):
-	"""Get a list of dict {"from_date": from_date, "to_date": to_date, "key": key, "label": label}
-		Periodicity can be (Yearly, Quarterly, Monthly)"""
-
-	fiscal_year = get_fiscal_year_data(from_fiscal_year, to_fiscal_year)
-	validate_fiscal_year(fiscal_year, from_fiscal_year, to_fiscal_year)
+def get_period_list(from_custom_date, to_custom_date, periodicity, accumulated_values=False, 
+		company=None, reset_period_on_fy_change=True):
+	"""
+	Returns a list of periods for which data needs to be generated based on periodicity or customized.
+
+	Args:
+		from_custom_date (datetype): [description]
+		to_custom_date (datatype): [description]
+		periodicity (int): [description]
```

Comment:
```suggestion
		from_custom_date (datetype): [description]
		to_custom_date (datatype): [description]
		periodicity (int): [description]
```
Either add a description or remove the placeholder